tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -543,9 +543,9 @@ ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_dr
 
 ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_tolerant:
```

Comment:
Please remove `Mellanox` from the skip condition of `ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_drop`. Thanks